### PR TITLE
Prevent semgrep output from breaking sarif output

### PR DESCRIPTION
**Pull request by brendongo**
This should solve https://github.com/returntocorp/semgrep-action/issues/346 I think the issue is that semgrep is printing out non sarif logging code.

### CODE REVIEW

**Comment by mschwager** [comment]
If you try the repro steps here does it fix the error? https://github.com/returntocorp/semgrep-action/issues/346#issue-962920821

Should we add `--quiet` to the `--json` path too?

**Comment by brendongo** [comment]
The json path actually writes the json to a file and reads that so does not need this. The correct way to do this for sarif would be to do that too but figured this was a quick fix

**Comment by brendongo** [comment]
Ugh no you're right it's definitely the --includes taking up too many characters. Gonna do this the right way then

**Comment by brendongo** [comment]
@mschwager the new changes run cleanly and produce a sarif file in the repo you linked in the issue

**Review by DrewDennison — APPROVED** [review]

File: src/semgrep_agent/semgrep.py [hunk 1]
```diff
@@ -258,12 +254,12 @@ def get_findings(
         # FIXME: This will crash when running on thousands of files due to command length limit
```
Comment (mschwager) [inline src/semgrep_agent/semgrep.py]: We can remove this `FIXME` now :+1:

**Review by mschwager — APPROVED** [review]
Nice!

**Pull request merged**